src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTests.vb

```diff
@@ -12850,5 +12850,34 @@ BC42104: Variable 'blah1' is used before it has been assigned a value. A null re
 </errors>)
 
         End Sub
+
+        <Fact>
+        <WorkItem(4196, "https://github.com/dotnet/roslyn/issues/4196")>
+        Public Sub BadDefaultParameterValue()
+            Dim source =
+<compilation>
+    <file name="a.vb">
+Imports BadDefaultParameterValue
+Module C
+    Sub Main
+        Util.M("test")
+    End Sub
+End Module
+    </file>
+</compilation>
+
+            Dim testReference = AssemblyMetadata.CreateFromImage(TestResources.Repros.BadDefaultParameterValue).GetReference()
+            Dim compilation = CompileAndVerify(source, additionalRefs:=New MetadataReference() {testReference})
+            compilation.VerifyIL("C.Main",
```

Comment:
@gafter not sure I understand why.  In this case the core issue is whether the compiler is using the appropriate default arguments or not.  Looking at the IL is the most direct way of verifying that.